sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CastSuite.scala

```diff
@@ -1018,4 +1019,24 @@ class CastSuite extends SparkFunSuite with ExpressionEvalHelper {
       checkEvaluation(ret, InternalRow(null))
     }
   }
+
+  test("SPARK-28023: Trim the string when cast string type to Boolean/Numeric types") {
```

Comment:
Since this is a kind of improvement, shall we remove `SPARK-28023: `?

Reply:
Removed.